docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1096 @@
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+-   Get the current language.
+
+    You can use the `CurrentLanguage` property of the [Tizen.Uix.VoiceControlManager.VoiceControlManagerClient](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.VoiceControlManagerClient.html) class.
+
+    If you want know language changes, you can use language change callback.
+
+    This function is not used when the voice control manager is in the `None` service status:
+
+    ```csharp
+    void GetCurrentLanguage()
+    {
+        try
+        {
+            string currentLanguage = VoiceControlManagerClient.CurrentLanguage;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+-   Get the microphone volume during recording.
+
+    You can use the `RecordingVolume` property of the [Tizen.Uix.VoiceControlManager.VoiceControlManagerClient](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.VoiceControlManagerClient.html) class.
+
+    The recording volume value is retrieved periodically with the short-term recorded sound data as decibels (dB).
+
+    The recording volume normally has a negative value, and 0 is the maximum value.
+
+    This function is used when the voice control manager is in the `Recording` service state:
+
+    ```csharp
+    void GetRecordingVolume()
+    {
+        try
+        {
+            float recordingVolume = VoiceControlManagerClient.RecordingVolume;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+-   Get or set the audio in type.
+
+    You can use the `AudioType` property of the [Tizen.Uix.VoiceControlManager.VoiceControlManagerClient](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.VoiceControlManagerClient.html) class.
+
+    The audio type values can be VC_AUDIO_ID_BLUETOOTH or VC_AUDIO_ID_MSF in string.
+
+    This function is used when the voice control manager is in the `Ready` state:
+
+    ```csharp
+    void GetSetAudioType()
+    {
+        try
+        {
+            /// Get
+            string audioType = VoiceControlManagerClient.AudioType;
+
+            /// Set 
+            VoiceControlManagerClient.AudioType = audioType;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+-   Get or set the recognition mode.
+
+    You can use the `RecognitionModeType` enumeration of the [Tizen.Uix.VoiceControlManager.RecognitionModeType](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.RecognitionModeType.html) enumeration as a parameter.
+
+    The default value of `RecognitionModeType` is `RecognitionModeType.StopBySilence`.
+    If you set the manual mode as `RecognitionModeType.Manual`.    
+    This function is used when the voice control manager is in the `Ready` state:
```

Comment:
@sungraejo Kindly add extra spaces between the lines for spacing in the output:

![image](https://user-images.githubusercontent.com/42992085/82068843-2aa51200-96f0-11ea-8d83-a6b647b27548.png)